frontend/src/components/stylists/scheduleManagementView.jsx

```diff
@@ -224,7 +224,7 @@ function ScheduleManagementView(props) {
       );
 
       // display blocks after successfull post...
-      if (responseData.id) {
+      if (responseData && responseData.id) {
```

Comment:
Both of these statements should be equivalent. If there is no responseData, responseDAta.id should evaluate to false.

Reply:
Should but failed once I tried it. Therefore the fix. It seems responseData was undefined which getting the id from undefined broke things. Again temporary fix to get stuff out the door. If you can find the core issue and fix it definetely do